tensorflow/core/kernels/data/parallel_interleave_dataset_op.cc

```diff
@@ -999,7 +999,8 @@ class ParallelInterleaveDatasetOp::Dataset : public DatasetBase {
             absl::make_unique<std::vector<Tensor>>(std::move(inputs));
         status = MakeIteratorFromInputElement(
             ctx_.get(), this, *element->inputs, element->id,
-            *instantiated_captured_func_, prefix(), &element->iterator);
+            *instantiated_captured_func_, prefix(), &element->iterator,
```

Comment:
Parallel interleave is more complicated with how `RecordStop` and `RecordStart` are used (i.e., in `SaveInternal`).